entrypoint.sh

```diff
@@ -112,8 +112,12 @@ fi
 
 git clone --branch ${DST_BRANCH} --single-branch --depth 1 https://${PERSONAL_TOKEN}@github.com/${DST_REPO}.git
 if [ "$?" -ne 0 ]; then
-    echo >&2 "Cloning '$DST_REPO' failed"
-    exit 1
+    echo >&2 "Cloning of '$DST_BRANCH' failed, creating the branch locally"
```

Comment:
Wouldn't it make sense to make this behaviour optional? What happens here is that if the cloning fails, it will always use the default branch, which might not be the inteded behaviour. I think there should be the default option to fail the execution completely, unless stated otherwise